charts/camunda-platform/test/unit/operate/golden/configmap.golden.yaml

```diff
@@ -24,6 +24,8 @@ data:
         host: camunda-platform-test-elasticsearch
         # Transport port
         port: 9200
+        # Elasticsearch full url
+        url: "http://camunda-platform-test-elasticsearch:9200"
```

Comment:
I'm not sure what the contents of the if statement would be? 
Are we checking if the user provides the port or not?
I remember when I was testing, the port was required for other apps.

Would you like me to create another url withou the port in the tpl file the does not include the port and then add it into the operate configmap? 
I don't see a problem in adding in the port since operate is working fine for me with the port added.

Reply:
According to the app config, no need to set both the `proto/host/port` and `url`.
https://github.com/camunda/zeebe/blob/7d7fec86461e993d75899266f95b89c5c2c2a23d/operate/config/application.yml

So we don't even need an if statement here, just remove the URL key.

Reply:
So this should be removed compeletly
```
        # Elasticsearch full url
        url: "http://camunda-platform-test-elasticsearch:9200"
```

Reply:
I removed the url entry in the operate configmap.

Is this only the case for operate or for other apps as well? (tasklist, optimize)

I see that tasklist has the url value in the application.yml so I will leave it: https://github.com/camunda/tasklist/blob/1acaf5afc76f21ad2ea43f2672c6178e8412a3ea/config/application.yml#L4

I can't tell if optimize requires the url parameter or not. This is the config I found for it: https://github.com/camunda/camunda-optimize/blob/fdd21395583cab2cc4bb866f32f1baebb39100d8/.ci/deployments-resources/argo/c8-optimize-application.yml#L4
I dont see an elasticsearch entry.

Reply:
I'm 99.9% sure tasklist uses the same format. (so you can remove the URL too).

Reply:
removed